gosdkexample/filtergvk/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/gosdk"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/gosdk/command"
+)
+
+// In this example, we use set a field in resources selecting by GVK.
+
+func main() {
+	cmd := command.AsMain(context.TODO(), mutateByGVK, command.Options{})
+	cmd.Short = "short description"
```

Comment:
I guess if I use the command then I need to provide these things but why would I do that?  As a consumer of the function SDK I am not going to really be calling the SDK directly, correct?

Reply:
Updated